index.js

```diff
@@ -108,7 +108,7 @@ exports.build = function build(argv, callback) {
         install = true;
         break;
       case 'b':
-        bundle = true;
+        console.error('Warning, the --bundle option has been removed');
```

Comment:
Sure. Wording doesn't matter much to me, this is just to soften the semver-major blow.